src/controllers/movies.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const router = express.Router();
```

Comment:
I think you should not need to have a router instance in your controller - might be worth looking at refactoring things so that anything to do with said instance is in your src/routers/movie.js file instead